### PR TITLE
[DEV-1516]: submit keys question form

### PR DESCRIPTION
**Pull request by MikeAtUqido**
#### List of Changes

* The question form should be submitted by pressing a combination of keys

#### Motivation and Context

The users expect to be able to submit a question pressing a combination of keys

#### How Has This Been Tested?

Manually

#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by jeremygordillo — APPROVED** [review]

File: .changeset/khaki-bags-call.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"nextjs-website": patch
+---
+
+Handle question form submit with a combination of keys
```
Comment (marcobottaro) [inline .changeset/khaki-bags-call.md:5]:
```suggestion
Handle webinar questions form submit with a combination of keys
```

File: apps/nextjs-website/src/components/organisms/WebinarQuestionsForm/WebinarQuestionsForm.tsx [hunk 2]
```diff
@@ -53,6 +53,14 @@ export const WebinarQuestionsForm = ({
     setQuestion(e.target.value);
   };
 
+  const handleKeyDown = (e: React.KeyboardEvent<HTMLInputElement>) => {
+    if (!question) return;
+    if (e.key === 'Enter' && (e.ctrlKey || e.shiftKey)) {
```
Comment (marcobottaro) [inline apps/nextjs-website/src/components/organisms/WebinarQuestionsForm/WebinarQuestionsForm.tsx:58]: I think you should explain why you chose this key combinations

**Review by marcobottaro — CHANGES_REQUESTED** [review]

File: apps/nextjs-website/src/components/organisms/WebinarQuestionsForm/WebinarQuestionsForm.tsx (hunk 2 above)
Comment (MikeAtUqido) [reply apps/nextjs-website/src/components/organisms/WebinarQuestionsForm/WebinarQuestionsForm.tsx]: The design team chose to use the `shift+enter` combination of keys

Comment (tommaso1) [reply apps/nextjs-website/src/components/organisms/WebinarQuestionsForm/WebinarQuestionsForm.tsx]: Maybe a comment on the function could be useful

**Review by tommaso1 — DISMISSED** [review]
I would add a comment on the keys

**Review by marcobottaro — APPROVED** [review]

**Pull request merged**